tests/src/Common/CoreFX/CoreFX.depproj

```diff
@@ -25,6 +25,7 @@
       <RuntimeIdentifier Condition="'$(OSGroup)' == 'Linux'">linux-x64</RuntimeIdentifier>
       <RuntimeIdentifier Condition="'$(OSGroup)' == 'OSX'">osx-x64</RuntimeIdentifier>
       <NugetRuntimeIdentifier>$(RuntimeIdentifier)</NugetRuntimeIdentifier>
+      <NoWarn>NU1603,NU1605</NoWarn>
```

Comment:
NU1605 could also be resolved by updating to the actual version resolved (this is a downgrade warning).